src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1235,7 +1235,7 @@ private function addExceptionsSection(ArrayNodeDefinition $rootNode)
                                 ->info('The status code of the response. Null to let Symfony decide.')
```

Comment:
Additionally, there is no possibility to explicitly specify `null` due to the validation (info message can be misleading).

Would you accept a fix for validation to allow this case?

Reply:
Indeed good catch, PR welcome.

Reply:
Here it is https://github.com/symfony/symfony/pull/46956